shell/packages/sandstorm-db/db.js

```diff
@@ -185,11 +185,13 @@ Contacts = new Mongo.Collection("contacts");
 //
 // Each contains:
 //   _id: random
-//   ownerId: The `_id` of the user who owns this contact.
-//   identityId:  The identity of the contacted user.
+//   ownerId: The accountId of the user account who owns this contact.
 //   petname: Human-readable label chosen by and only visible to the owner. Uniquely identifies
 //            the contact to the owner.
 //   created: Date when this contact was created.
+//   identityId: The `_id` of the user whose contact info this contains.
+//   profile: An object containing denormalized profile data for this contact. See the `Users`
```

Comment:
I'm tempted to try to figure out how to avoid adding this `profile` field. I remember you were thinking about implementing a join at some point. Did you run into any problems with that approach?